frontend/views/containers/payments/PaymentDetail.vue

```diff
@@ -114,10 +112,13 @@ export default ({
         sbp('okTurtles.events/emit', SET_MODAL_QUERIES, 'PaymentDetail', { id })
       }
       if (payment) {
-        this.payment = cloneDeep(payment)
+        const periodstamp = await this.historicalPeriodStampGivenDate(payment.meta.createdDate)
```

Comment:
Note: I'm not sure why this was saved to a variable that's never used... in the previous commit it was being inlined on the line below and that's fine. Won't hold up the PR thought

Reply:
It's used to compute `isOldPayment`, so I had to save it to a new variable to avoid duplication

Reply:
Ah good point! I didn't see that! 😆